### ISSUE TITLE
Snapturn for controller

### ISSUE THREAD

**Issue by Eispfogel** [DrBeef/QuakeGVR#33]
It would be nice, if we could rotate the view by pressing left/right on the touchpad.
Sometimes...you just cannot rotate that much on a couch ;)

For now i have to use an Xbox One Gamepad for that, but since there is no deadzone for the sticks it keeps on moving me all over the place.

Edit: I think i need to specify this more - Swiping should rotate the player and clicking/touching left/right should move me in that direction.

**Comment by neitey** [comment]
A work around solution is to use a cheap BT Android Gamepad. I get drift
movement with xbox one but not with the two Android controllers i have.

Eispfogel <notifications@github.com> schrieb am Fr., 15. März 2019, 20:43:

**Comment by Eispfogel** [comment]
Yeah this is a workaround! But not for the Snapturning on the 3DoF controller ;)

**Comment by neitey** [comment]
Second that. Snapturn option for the touchpad is needed.

**Comment by QUAKEULUS** [comment]
Gesture based maybe? Like tilt controller left over 50 degrees and press on left edge to snap turn 30 degrees left, hold to keep turning?
